sharktank_models/test_suite/ireers_tools/artifacts.py

```diff
@@ -0,0 +1,234 @@
+# Copyright 2023 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+from typing import Any, Callable, Collection, Dict, Union
+import functools
+from pathlib import Path
+from tqdm import tqdm
```

Comment:
This looks unused, could remove then add back if it does get used somewhere.

https://tqdm.github.io/ (it wraps iterables to low a progress meter)